android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -96,6 +96,15 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         String startTime = ISO8601Date.get12HourTime(ISO8601Date.getDateObject(currentSession.getStartTime()));
         String endTime = ISO8601Date.get12HourTime(ISO8601Date.getDateObject(currentSession.getEndTime()));
 
+        /*We are following 12 hour format so changing 00 PM to 12 PM */
```

Comment:
No need to manipulate string manually. You can just change 'KK' to 'hh'  in get12HourTime function in ISO8601Date.java
KK  is for 0-11 am/pm
hh is for 1-12 am/pm

Reference : https://developer.android.com/reference/java/text/SimpleDateFormat.html